lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -1,5 +1,6 @@
 require 'pathname'
 require 'json'
+require_relative '../../../puppet_x/puppetlabs/powershell_manager'
```

Comment:
I think require_relative is only available in ruby 1.9, does the DSC module require a specific puppet or ruby version so this doesn't blow up if someone was to (unlikely) run on puppet < 3.2.0